pyproject.toml

```diff
@@ -77,18 +77,19 @@ torch = {version = ">=1.6.0, <3.0.0", optional = true}
 pytorch-lightning = {version = ">=1.6.0, <3.0.0", optional = true}
 
 ipywidgets = {version = ">=7.7,<8.2", optional = true}
+plotly = {version="^5.22.0", optional = true}
 
 
 [tool.poetry.extras]
 lightfm = ["lightfm"]
 nmslib = ["nmslib"]
 torch = ["torch", "pytorch-lightning"]
-visuals = ["ipywidgets"]
```

Comment:
I've just realised that we need to add `MetricsApp` to compat.py https://github.com/MobileTeleSystems/RecTools/blob/main/rectools/compat.py and test https://github.com/MobileTeleSystems/RecTools/blob/main/tests/test_compat.py